src/vaadin-avatar.html

```diff
@@ -7,29 +7,51 @@
 <link rel="import" href="../../polymer/polymer-element.html">
 <link rel="import" href="../../vaadin-themable-mixin/vaadin-themable-mixin.html">
 <link rel="import" href="../../vaadin-element-mixin/vaadin-element-mixin.html">
+<link rel="import" href="vaadin-avatar-icons.html">
 
 <dom-module id="vaadin-avatar">
   <template>
     <style>
       :host {
         display: inline-block;
+        border-radius: 50%;
+        overflow: hidden;
+        height: 44px;
+        width: 44px;
+        font-family: 'vaadin-avatar-icons';
+      }
+
+      img {
+        height: 100%;
+        width: 100%;
       }
 
       :host([hidden]) {
         display: none !important;
       }
+
+      svg {
+        font-size: 5.6em;
+      }
+
+      svg#avatar-icon > text::before {
+        font-family: 'vaadin-avatar-icons';
+      }
     </style>
-    <slot></slot>
+    <img hidden$="[[!__imgVisible]]" src$="[[img]]" />
+    <svg hidden$="[[__imgVisible]]" id="avatar-icon" viewBox="-50 -50 100 100" preserveAspectRatio="xMidYMid meet">
```

Comment:
Looks like `<svg>` element does not support `hidden` attribute out of the box, at least it's not hidden in Chrome.

Adding the style for it explicitly should help:
```css
[hidden] {
  display: none !important;
}
```

Reply:
Good catch! Updated.